buildscripts/patches/openmw/0015-water-refraction-dirty-patch.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/files/shaders/water_fragment.glsl b/files/shaders/water_fragment.glsl
+index 86edd8698..c49440e2a 100644
+--- a/files/shaders/water_fragment.glsl
++++ b/files/shaders/water_fragment.glsl
+@@ -213,6 +213,7 @@ void main(void)
+ #endif
+ 
+     vec2 screenCoordsOffset = normal.xy * REFL_BUMP;
++/*
```

Comment:
Same as above- can this be fixed in openmw?

Reply:
nope i think its limitation of gl4es and opengl es "depth buffer cant be read", so that refractiondepthmap part in shaders doesnt work properly, this may make it look little better.

Reply:
its just workaround, so it dont look that broken. May be removed too.